orgs/opentelekomcloud-blueprints/people/members.yml

```diff
@@ -0,0 +1,2 @@
+---
+users: []
```

Comment:
Why no users?

Reply:
no users in this organization, idk why

Reply:
Maybe you don't have enough permissions and they are hidden?